pycytominer/cyto_utils/cells.py

```diff
@@ -452,8 +454,7 @@ def load_compartment(self, compartment):
             feats[i] = row[num_meta:]
 
         # Return concatenated data and metainformation of compartment
-        return pd.concat(
-            [metas, pd.DataFrame(columns=feat_cols, data=feats)], axis=1)
+        return pd.concat([metas, pd.DataFrame(columns=feat_cols, data=feats)], axis=1)
```

Comment:
this looks like a format update. Is this what black recommends for the file? (we use black)

Reply:
Yes, this ended up being what Black recommended from my local development environment (VS Code autoformat on save, terminal-based `black --check ...` shows all good). It may be helpful at some point to consider the use of [Black configuration specifications within a pyproject.toml file](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file) to make sure local linting / formatting changes are consistent (especially when it comes to Python or Black version differences). We could also pull the Black (or other linting) Github Action checks from the `develop` branch into `main/master` for greater assurances here.

Reply:
Gotcha! Yes, we'll want to standardize. Likely outside scope of the present PR.

Reply:
Sounds good, I opened #230 to help address this in the future.